src/libs/Navigation/AppNavigator/createCustomStackNavigator/index.js

```diff
@@ -60,15 +59,12 @@ function ResponsiveStackNavigator(props) {
     });
 
     const stateToRender = useMemo(() => {
-        const {reportRoutes, rhpRoutes, otherRoutes} = splitRoutes(state.routes);
-
-        // Remove all report routes except the last 3. This will improve performance.
```

Comment:
We can keep this comment but just move to [here](https://github.com/Expensify/App/pull/31014/files#diff-2d96003414d05766e96a83bb4843e25274674656f66a9ec541727ad585e82f5dR34)